autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/BasicSelect/BasicSelect.tsx

```diff
@@ -44,6 +55,7 @@ const BasicSelect: React.FC<BasicSelectProps> = ({ id }) => {
       <FormLabel>Options</FormLabel>
       <table className={styles.tableContainer}>
         <tbody>
+          <BasicCheckbox id={id} value="includeFull" label="Include Full Sections" />
           {hasHonors
             ? <BasicOptionRow id={id} value="honors" label="Honors" />
             : null}
```

Comment:
I'd appreciate if you could rewrite this to only have one table (instead of creating an identical table for the "there are no honors or remote sections" text and another if there are honors/remote/async sections). Instead of returning the entire table if there are no honors/remote sections, determine the correct thing to render (either the `BasicOptionRow`s or the placeholder text) and store it in a variable